Tutorials/2024_PNNL_Atmospheric_Course/6-ACT-TRACER-Dust.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    display.plot('signal_return_cross_pol', cb_friendly=True, vmin=-20, vmax=20)
</pre><p>These calls should be cvd_friendly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-Development/ARM-Notebooks/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>